### ISSUE TITLE
Nested toggles

### ISSUE THREAD

**Issue by balex25** [kommitters/editorjs-toggle-block#225]
### Description

Use this nested toggle as predefined blocks:
```
    data: {
      blocks: [
        {
          type: "paragraph",
          data: {
            text: "Your text..."
          }
        },
        {
          type: "toggle",
          data: {
            fk: "fk-1",
            items: 4,
            status: "open",
            text: "Parent toggle"
          }
        },
        {
          type: "paragraph",
          data: {
            text: "Parent toggle item 1"
          }
        },
        {
          type: "paragraph",
          data: {
            text: "Parent toggle item 2"
          }
        },
        {
          type: "paragraph",
          data: {
            text: "Parent toggle item 3"
          }
        },
        {
          type: "toggle",
          data: {
            fk: "fk-2",
            items: 1,
            status: "closed",
            text: "Parent toggle item 4"
          }
        },
        {
          type: "paragraph",
          data: {
            text: "Child 1"
          }
        },
        
      ]
    }
```

Output:
![08BH3MC](https://i.imgur.com/08BH3MC.png)

How should it be:
![u6LWaMV](https://i.imgur.com/u6LWaMV.png)

**Parent toggle item 4** should be inside **Parent toggle**, but now it rended outside of the parent toggle.

### Current behavior

1. Use the data JSON above.
2. Try to load nested toggle.

### Expected behavior

Items nr. 4 should be inside of the parent toggle, as on data items are set **4**, so the _next 4 blocks should be the child of the parent toggle_. this works fine with any block type but seems to _not work when the last block child is a toggle type._

**Comment by miguelnietoa** [comment]
Hello @balex25,

Thank you for bringing this to our attention! We appreciate your input.

Just a heads up, the nested toggles feature is currently not supported. Rest assured, we've taken note of the behavior you've reported, and it will be considered in the process of supporting this new feature.

We are closing this issue as a duplicate of https://github.com/kommitters/editorjs-toggle-block/issues/170.

Feel free to contribute to the discussion on the linked issue if you have any additional insights.
